com.github.alainm23.planner.json

```diff
@@ -0,0 +1,89 @@
+{
+  "app-id": "com.github.alainm23.planner",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.34",
+  "sdk": "org.gnome.Sdk",
+  "base": "io.elementary.BaseApp",
+  "base-version": "juno-19.08",
+  "command": "com.github.alainm23.planner",
+  "finish-args": [
+    "--share=ipc",
+    "--device=dri",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--talk-name=org.gtk.vfs.*",
+    "--talk-name=org.evolution.dataserver.*",
```

Comment:
This assumes you need everything from eds; is it the case? you can be a bit more specific here. Run the application with 
`--log-session-bus` to get a list of the dbus names it tries to communicate with.